.terraform-docs.yml

```diff
@@ -2,3 +2,6 @@ output:
   file: README.md
   mode: inject
 formatter: "markdown"
+recursive:
+  enabled: true
+  path: gcp-gke
```

Comment:
Should we follow the pattern that we documented in notion and Poom started with CloudSQL module and put them under a `modules/` folder for consistency?  That way this `.terraform-docs.yaml` can be the same across all repos too.

Reply:
yeah ok